CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Hosting a Bitcoin Core review club meeting
+
+## General Tips
+
+Some tips for hosting a review club meeting:
+
+- try to post notes and questions on the Friday the week before the meeting.
+  That gives attendees time to review the PR and prepare for the meeting.
+- prepare a range of questions from basic to advanced. The review club is a way
+  to help new developers develop. There should be something for everone.
+- come with the mindset that the attendees are people who are as smart as you,
+  but don't have the context that you do in Bitcoin protocol development.
+- very open questions to the group (eg "Does everyone know how tx gossip works
+  in Bitcoin?") sometimes fall flat because all the attendees are waiting for
+  someone else to answer. Instead try to make the questions focussed on the
```

Comment:
nit: focused

Reply:
In British/Australian/New Zealander English, it's not an error AFAIK. https://www.collinsdictionary.com/dictionary/english/focussed

Reply:
TIL